### ISSUE TITLE
[syncfusion_flutter_pdfviewer] Exception: Invalid image data

### ISSUE THREAD

**Issue by scuti-dai** [syncfusion/flutter-widgets#1124]
![Screenshot 2023-03-14 at 14 32 48](https://user-images.githubusercontent.com/104118833/224927599-0e78bf70-2557-4648-8ffa-9798cd34ea89.png)
[✓] Flutter (Channel stable, 3.7.7, on macOS 13.0 22A380 darwin-arm64, locale en)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 14.0.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2022.1)

**Comment by deepika134** [comment]
Hi @scuti-dai ,We suspect that there is a problem with loading the Pdf in SfPdfViewer from the provided stack race. We tried to reproduce the reported issue, but the Pdf document is loaded properly without any exception in our end. We have shared the tested sample for your reference, and which can be downloaded from the attachment. We suspect it may be a document specific issue, kindly share the Pdf document and code snippet or issue reproducing steps. This will be helpful for us to provide a prompt solution at the earliest.
[github_446871.zip](https://github.com/syncfusion/flutter-widgets/files/10981038/github_446871.zip)
